omod/src/main/web/dashboardwidgets/obsacrossencounters/obsacrossencounters.controller.js

```diff
@@ -12,6 +12,7 @@ export default class ObsAcrossEncountersController {
     this.simpleEncs = [];
     this.headers = [];
     this.openmrsRest.setBaseAppPath("/coreapps");
+    this.isArray = angular.isArray;
```

Comment:
```suggestion
```

Reply:
What's the deal with this?

Reply:
Did this to be able to use `angular.isArray` on the html by referring to `$ctrl.isArray`:
https://github.com/openmrs/openmrs-module-coreapps/pull/412/files#diff-a90b523909e4c003ad9991c2d71f6106ca227964ed989c03bfe2589c2dcb6e21R41

Reply:
Oh yeah, I see that call now, thanks!